docs/i18n.md

```diff
@@ -24,25 +26,28 @@ Don't use class components, use [hooks](https://reactjs.org/docs/hooks-overview.
 
 ```jsx
 export function MyComponent() {
-  const { t } = useTranslation('myNamespace');
+  const { t } = useTranslation('currentModule');
 
   return <div>{t('I am a React component!')}</div>;
 }
 ```
 
 ### 3. If your strings include html, wrap them in [`Trans`](https://react.i18next.com/components/trans-component) component
 
+_Note: Please provide the current namespace to `Trans` component by passing `ns="currentModule"` and `t={t}` in props.
+@TODO remove the `ns` part when gilbsgilbs/babel-plugin-i18next-extract#153 gets resolved.)_
+
 ```jsx
 import { Trans, useTranslation } from 'react-i18next';
 
 export function MyComponent() {
-  const { t } = useTranslation('myNamespace');
+  const { t } = useTranslation('currentModule');
 
   return (
     <>
       <div>{t('I am a text to translate!')}</div>
       <div>
-        <Trans>
+        <Trans ns="currentModule" t={t}>
```

Comment:
We only need `t={t}` and not `ns="currentModule"` I think, from reading https://react.i18next.com/latest/trans-component`

Reply:
Added both `ns` and `t` to every component, with TODO about removing `ns` when the bug in babel-plugin-i18next-extract library is removed. #1368 
See [below](https://github.com/Trustroots/trustroots/pull/1369#discussion_r406729506) for further discussion.